channels/http.py

```diff
@@ -30,11 +30,10 @@ class AsgiRequest(http.HttpRequest):
     # body and aborts.
     body_receive_timeout = 60
```

Comment:
What are we going to do about it? Why do we need `body_receive_timeout`?

Reply:
It used to make sense https://github.com/django/channels/commit/920882f1da3e3c9a9432f2172bf06700bfc6fa48